packages/common/src/dto/api/index.ts

```diff
@@ -116,6 +116,10 @@ export interface IServerConfig {
     plugins: IWorkspacesDefaultPlugins[];
     pvcStrategy: string | undefined;
   };
+  editorSelector?: {
```

Comment:
I would suggest to rename this field, because it's about tuning the visibility of editors (showing or hiding some of them).

```suggestion
  editorsVisibility?: {
```

Reply:
I have fixed it.